test/simplifyExpression/arithmeticSearch/arithmeticSearch.test.js

```diff
@@ -6,11 +6,11 @@ function testArithmeticSearch(exprStr, outputStr) {
   TestUtil.testSimplification(arithmeticSearch, exprStr, outputStr);
 }
 
-describe.skip('evaluate arithmeticSearch', function () {
+describe('evaluate arithmeticSearch', function () {
   const tests = [
     ['2+2', '4'],
     ['2*3*5', '30'],
-    ['9/4', '9/4'], //  does not divide
+    ['9/4', '9 / 4'], //  does not divide
   ];
```

Comment:
Ah, yeah there's probably fewer tests because arithmetic was the first thing we did and most of those tests are in simplifyExpression from before we had more modular tests. I agree we should add some more here - would be easier to understand what this treeSearch does

- unary minus and implicit multiplication: what kind of things are you thinking of?
- applying roots are handled in a separate root-specific code, and similar for absolute value (since they're function nodes and not operator nodes))